lib/turn.rb

```diff
@@ -0,0 +1,25 @@
+
+class Turn
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if card.answer == @guess
+      true
+    else
+      false
+    end
```

Comment:
since `==` will automatically return true or false, you could simplify this method to just have `@card.answer == @guess`. Notice, I'm using the instance variable for card instead of calling to the method card (defined by setting your attr_reader). When you can, call to the instance variable instead of the attr_Reader, because your attr_reader is basically returning the instance variable anyway -- cut out the middle man where you can to help optimize your code. It is ruby convention to call to instance variables instead of their attr reader when in the calss where the attributes are defined.